cmake/modules/OmrDetectSystemInformation.cmake

```diff
@@ -202,12 +202,18 @@ macro(omr_detect_system_information)
 			# In CMake 3.14 and prior, XLClang uses CMAKE_C_COMPILER_ID "XL"
 			# In CMake 3.15 and beyond, XLClang uses CMAKE_C_COMPILER_ID "XLClang"
 			set(_OMR_TOOLCONFIG "xlc")
+			message(STATUS "CMAKE_C_COMPILER is ${CMAKE_C_COMPILER}")
 			if(CMAKE_C_COMPILER MATCHES ".*xlclang$")
 				# Checking the CMAKE_C_COMPILER command is necessary to determine if XLClang is
 				# the compiler, since XLClang might have CMAKE_C_COMPILER_ID "XL" or "XLClang"
 				# depending on the CMake version. Without this check, it's ambiguous whether the
 				# compiler is XLC or XLClang.
 				set(CMAKE_C_COMPILER_IS_XLCLANG TRUE CACHE BOOL "XLClang is the C compiler")
+			elseif(CMAKE_C_COMPILER MATCHES ".*ibm-clang.*")
+				set(CMAKE_C_COMPILER_IS_OPENXL ON CACHE BOOL "ibm-clang is the C compiler")
+				set(_OMR_TOOLCONFIG "openxl")
+			else()
+				message(STATUS "NO MATCH")
```

Comment:
Hi Keith, I was verifying all my cleanup changes with different builds. After making the above mentioned changes, while Open XL passes with no problems, XLC build gives a fatal error, because I guess it is entering thiis z/OS section, but not matching with the other xlclang compilers and in place of the "NO MATCH" its just throwing the error.

Perhaps it makes more sense to update this to 
`message(STATUS "NO XLClang match. Using xlc toolchain.")`

Does this sound good?